staging/src/k8s.io/api/go.mod

```diff
@@ -11,6 +11,7 @@ require (
 )
 
 replace (
+	golang.org/x/net => golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
```

Comment:
since this is no longer the preferred version, add a `// pinned to release-branch.go1.13` comment to the root go.mod to explain why

Reply:
Done.